frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -340,6 +346,7 @@ export const createOrUpdateDeployment = (
     'alpha.image.policy.openshift.io/resolve-names': '*',
     ...getTriggerAnnotation(name, imageName, namespace, imageChange),
   };
+  const imageStreamLabels = imageStream && imageStream.metadata.labels;
```

Comment:
It is not good to copy all the ImageStream labels to the deployment. It would be nice to copy the label that we need. Like here we need only the `app.openshift.io/runtime` label. And one thing I noticed is that `defaultLabels` have `app.openshift.io/runtime` so can we fix `getAppLabels` so that it adds the correct value to `app.openshift.io/runtime` label?

Reply:
Thanks @vikram-raj. I have now extracted the exact Label value and have passed it as the `runtimeIcon` in the `getAppLabels`.